.changeset/smooth-rats-jog.md

```diff
@@ -0,0 +1,5 @@
+---
+"single-spa-foundry-worker": patch
```

Comment:
```suggestion
"single-spa-foundry-worker": minor
```

The cache-control header is a new feature, not a bug fix, so it should be a minor bump.